internal/abci/accumulator.go

```diff
@@ -165,14 +165,14 @@ func (app *Accumulator) InitChain(req abci.RequestInitChain) abci.ResponseInitCh
 		Height:   -1,
 	})
 
-	err = app.chain.CheckTx(tx)
-	if err != nil {
-		panic(fmt.Errorf("failed to validate genesis TX: %v", err))
+	customErr := app.chain.CheckTx(tx)
+	if customErr != nil && customErr.Code != protocol.CodeOK {
```

Comment:
Unless I'm missing something, the executor never returns `CodeOK`, so we don't need this check.

Reply:
This is if in case in future somebody returns CodeOk, then also it will work